examples/SampleApp/src/screens/ChannelScreen.tsx

```diff
@@ -147,6 +147,13 @@ export const ChannelScreen: React.FC<ChannelScreenProps> = ({
         keyboardVerticalOffset={Platform.OS === 'ios' ? 0 : -300}
         messageId={messageId}
         NetworkDownIndicator={() => null}
+        onPressMessage={({ additionalInfo, defaultHandler, emitter }) => {
+          if (emitter === 'textMention') {
+            console.log(additionalInfo?.user);
+            return;
+          }
+          defaultHandler?.();
+        }}
```

Comment:
remove it from sample app, if there is no UX for this. Console.logs are not visible to end users

Reply:
✅